src/api/core/ProxyResultBuilder.ts

```diff
@@ -0,0 +1,41 @@
+import { ProxyResult } from "aws-lambda";
+
+export default class ProxyResultBuilder {
+
+    protected response: Partial<ProxyResult> = {};
+
+    constructor(){
+        this.response.headers = {};
+    }
+
+    public status(status: number): ProxyResultBuilder {
+        this.response.statusCode = status;
+        return this;
+    }
+
+    public body(body: string | object): ProxyResultBuilder {
```

Comment:
@bigLucas 
So far I didn't understand why the functions returns its own class. Still looking for.

Reply:
@Aristevao 
This class is a builder, a kind of design pattern, you can search for this to understand more about it.

Reply:
@bigLucas 
Thanks, I will read about this design pattern.